src/scss/fields.scss

```diff
@@ -1,98 +1,63 @@
-@include oFormsClass(o-forms__field-group) {
+@if not $o-forms-is-silent {
+.o-forms-group {
```

Comment:
Only partially supporting silent mode is kind of weird. I'm not very sure that's a good idea. @triblondon What do you think?